test/common/run-tests

```diff
@@ -113,23 +114,25 @@ def finish_test(opts, test, affected_tests):
             proc = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
             changed = proc.communicate(test.output)[0]
             if proc.returncode == 0:
-                if test.output != changed:
-                    changed += b"\n"
```

Comment:
Ha,  I can put this back in, because `breaklines` behaves a bit differently than `split("\n")` in the metter of trailing newlines:
```py
>>> x = "foo\nbar\n"
>>> print(x.splitlines()[-1])
bar
>>> print(x.split("\n")[-1])

>>> 

```

Reply:
Done